app/controllers/lead_providers/openapi_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 # source: https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/controllers/api_docs/vendor_api_docs/openapi_controller.rb
 
+require "lead_provider_api_specification"
+
```

Comment:
Why do we need it here?

Reply:
prior to this `lead_provider_api_specification` is loaded by `app/controllers/lead_providers/guidance_controller.rb`. If that file is never loaded then `lead_provider_api_specification` will also not be available. `/lib` is not autoloaded by rails therefore must be required by hand before it is available

Reply:
from `config/environments/test.rb` `config.eager_load = false` therefore not everything is loaded prior to test suite running.

Reply:
Ah, alright, that makes sense